Model with contradicting Constraint (#276)
eclipse-esmf/esmf-semantic-aspect-meta-model: documentation/modules/ROOT/pages/characteristics.adoc

| `samm-c:minValue` | The minimum length. Must be given as `xsd:nonNegativeInteger`. | {nok}
|===

IMPORTANT: When using the `samm-c:minValue` parameter in Length Constraint, it is crucial to consider the following:

> * Creating a Model with a <<set-characteristic>>/<<sorted-set-characteristic>> based on a finite Base Characteristic (`https://eclipse-esmf.github.io/samm-specification/snapshot/characteristics.html#boolean-characteristic[Boolean]` or `https://eclipse-esmf.github.io/samm-specification/snapshot/characteristics.html#enumeration-characteristic[Enumeration]`)

Reviewer: We should always use internal refs or xrefs instead of hard-coded URLs when linking inside the docs:

```suggestion
Creating a Model with a <<set-characteristic>>/<<sorted-set-characteristic>> based on a finite Base Characteristic (<<boolean-characteristic>> or <<enumeration-characteristic>>)
```

> and adding Length Constraint, the `samm-c:minValue` should not exceed the total number of elements in the Base Characteristic.

Reviewer: ```suggestion
and adding Length Constraint, the `samm-c:minValue` must not exceed the total number of elements in the Base Characteristic.
```

> Models with such conditions are not recognized as invalid, but for which no payload can exist.

Reviewer: ```suggestion
Models with such conditions are not recognized as invalid, but no valid payloads can exist for them.
```

> The responsibility is borne by the user who created these models with such Length Constraints for the characteristics.

Reviewer: I would remove this sentence completely
